controller/index.js

```diff
@@ -1,16 +1,30 @@
 
 var pkg = require('../package');
+var markdown = require('markdown').markdown;
+var fs = require('fs');
 
 function setup(){
 
   function controller( req, res, next ){
 
-    // stats
-    res.json({
-      name: pkg.name,
-      version: {
-        number: pkg.version
+    fs.readFile('./DOCS.md', 'utf8', function (err, content) {
+      if (!err) {
+        var header = '# Pelias API\n';
+        var version = '### Version: ['+ pkg.version+ '](https://github.com/pelias/api/releases)\n';
+        var style = '<style>html{font-family:monospace}</style>';
+        
+        res.send(style + markdown.toHTML(header + version + content));
+
+      } else {
```

Comment:
I'm a little uneasy about serving `text/html` sometimes and `application/json` at other times. It might be nicer to check the `Accepts` header from the User Agent and serve content accordingly. Browsers will always send a header such as `Accept:text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8` when they want HTML and client libraries will always send 'Accept:application/json' so no-one gets nasty surprises.